charts/govuk-apps-conf/rules/router/README.md

```diff
@@ -0,0 +1,5 @@
+## Run Prometheus unit tests for alerts
```

Comment:
I would have preferred a separate helm chart to include all the Prometheus alerts.
Usually, this chart is for shared config between apps.

I would rename `rules` to `prometheus-rules` if kept in this chart and move the README.md file directly in that folder as I guess it applies to multiple apps that we will add here.

Reply:
I think some automated test should be run in GitHub Actions for that, might be a separate story.

Reply:
We were deciding on whether to relocate the prometheus rules to either rails-app or apps-conf, we ultimately chose the latter as the structure made more sense, as we could group alerts with their respective apps, this also provided an easier way to deploy using ArgoCD as the rules would deploy alongside its app.
As opposed to configuring automation for an entirely new helm chart.